<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="10" total="12" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-07-29T16:38:38 EET" name="Default suite" finished-at="2025-07-29T16:38:55 EET" duration-ms="16466">
    <groups>
    </groups>
    <test started-at="2025-07-29T16:38:38 EET" name="Default test" finished-at="2025-07-29T16:38:55 EET" duration-ms="16466">
      <class name="MyTestCases">
        <test-method is-config="true" signature="myTestSetup()[pri:0, instance:MyTestCases@260e86a1]" started-at="2025-07-29T16:38:38 EET" name="myTestSetup" finished-at="2025-07-29T16:38:44 EET" duration-ms="5656" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myTestSetup -->
        <test-method signature="MouseHover()[pri:0, instance:MyTestCases@260e86a1]" started-at="2025-07-29T16:38:44 EET" name="MouseHover" finished-at="2025-07-29T16:38:49 EET" duration-ms="4669" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MouseHover -->
        <test-method signature="Calender()[pri:3, instance:MyTestCases@260e86a1]" started-at="2025-07-29T16:38:49 EET" name="Calender" finished-at="2025-07-29T16:38:55 EET" duration-ms="6050" status="FAIL">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Keys to send should be a not null CharSequence]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:96)
at MyTestCases.Calender(MyTestCases.java:280)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Calender -->
      </class> <!-- MyTestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
